index.bs

```diff
@@ -291,60 +291,60 @@ Note: This algorithm manages the fetching of a [=/background fetch record=]. One
     Note: If the |rangeStart| is 0, a normal request is made. This allows the initial request to make use of content encoding, since `Accept-Encoding: identity` is added to requests with a range header.
 
   1. Let |fetchAttemptComplete| be false.
-  1. Let |lastTransmittedSize| be 0.
-  1. [=/Fetch=] |request|.
-
-    Issue: The remainder of this step uses fetch "callbacks" which currently queue tasks. This isn't desirable or possible here, so let's pretend that tasks aren't queued. ([issue](https://github.com/whatwg/fetch/issues/536#issuecomment-330184276))
-
-    To [=process request body=] for |request|, run these steps:
-      1. Let |transmittedSize| be |request|'s [=request/body=]'s [=body/transmitted bytes=].
-      1. Increment |bgFetch|'s [=background fetch/uploaded=] by |transmittedSize| minus |lastTransmittedSize|.
-      1. Set |lastTransmittedSize| to |transmittedSize|.
-      1. [=Update background fetch instances=] for |bgFetch|.
-
-    To [=process response=] for |response|, run these steps:
-      1. If |response| is a [=network error=], then:
-        1. If the resource is [=temporarily unavailable=] and |request|'s [=request/method=] is \``GET`\`, then wait until the resource is not [=temporarily unavailable=], then set |fetchAttemptComplete| to true and abort these steps.
-
-          Note: If |request|'s [=request/method=] is not \``GET`\`, reissuing the request may have unwanted side effects. If a standard method to resume requests becomes available, it'll be adopted here.
-
-        1. If |response| is an [=aborted network error=], then set |responseData|'s [=background fetch response/result=] to `"aborted"`, otherwise `"fetch-error"`.
-        1. Set |fetchAttemptComplete| to true, and abort these steps.
-      1. If |response|'s [=response/status=] is `206`, then:
-        1. If [=validate a partial response=] for |rangeStart|, |response|, and |responseData|'s [=background fetch response/response=] returns invalid, then:
-          1. Set |responseData|'s [=background fetch response/result=] to `"fetch-error"`.
-          1. Set |fetchAttemptComplete| to true.
-          1. [=fetch/Terminate=] the ongoing fetch, and abort these steps.
-      1. Otherwise:
-        1. Set |responseData|'s [=background fetch response/result=] to `"redundant"`.
-        1. Set |responseData| to a new [=background fetch response=].
-        1. Set |record|'s [=background fetch record/response data=] to |responseData|.
+  1. [=/Fetch=] |request| with the following arguments:
+    <dl class="compact">
+      : [=fetch/useParallelQueue=]
+      :: True.
```

Comment:
@annevk I know the fetch spec says I shouldn't need this, but my intent is to be able to do this processing without any page open.

Reply:
The tricky aspect is how we enforce any of the things that Fetch requires a client for, such as referrer and CSP. It seems you would have to obtain these ahead of time and pass them in somehow, but there's no infrastructure for that as of yet.